packages/template-retail-react-app/app/components/drawer-menu/index.jsx

```diff
@@ -114,49 +124,56 @@ const DrawerMenu = ({isOpen, onClose = noop, onLogoClick = noop, root}) => {
 
                     {/* Main Content */}
                     <DrawerBody>
-                        {showLoading && <LoadingSpinner />}
+                        <div
+                            id="category-nav"
+                            aria-live="polite"
+                            aria-busy="true"
+                            aria-atomic="true"
+                        >
```

Comment:
Was this tested via https://www.perkins.org/resource/mac-voiceover/ or similar to verify it works as expected for screen readers?

Reply:
Yes! Just tested and the screen reader announces the categories and the `aria-busy` attribute is correctly set to false after everything is loaded